crossfit/backend/torch/op/base.py

```diff
@@ -81,27 +81,41 @@ def call(self, data, partition_info=None):
             if self.post is not None:
                 output = self.post(output)
 
-            all_outputs_ls.append(output)
-
+            if self.model.string_tok_inf:
```

Comment:
Why don't you just check if it's a `CT2CustomModel` (or check if it's a `TranslationConfig`, whichever makes more logical sense to you) instead of creating the `string_tok_inf` field?

Reply:
Modified.